builder/Cargo.toml

```diff
@@ -14,6 +14,7 @@ telemetry = []
 data-pipeline = []
 symbolizer = []
 library-config = []
+ddcommon = []
```

Comment:
No need to add ddcommon to the features list because ddcommon module is always added to the module list.

Reply:
removed in https://github.com/DataDog/libdatadog/pull/985/commits/df8d0ae1b9cf4d7b69a72b3bb5ad54d8174ecc2f